Assets/Scripts/ScriptableObjects/Level.cs

```diff
@@ -0,0 +1,13 @@
+using UnityEngine;
+
+[CreateAssetMenu(fileName = "NewLevel", menuName = "Scene Data/Level")]
+public class Level : GameScene
+{
+    public enum World {MEADOW,TUNDRA,BEACH}
+    //Settings specific to level only
+    [Header("Level specific")]
+    public int numberOfEnd;
+    public World world;
+    public int cameraSize;
+    public Color LightColor;
```

Comment:
```suggestion
    public Color lightColor;
```

Worth to be noted that most use PascalCase for class attributes and camelCase for local variables, on Unity. But since we did not follow it, let's keep it consistent